### ISSUE TITLE
explain shader *_PIXEL_SIZE better

### ISSUE THREAD

**Issue by razcore-art** [godotengine/godot-docs#1538]
For example for `TEXTURE_PIXEL_SIZE` it says *Default 2D texture pixel size.* which I guess it's fine, but thought it means a `vec2` with the size of the texture in pixels say `vec2(64.0, 64.0)` for the default `icon.png` asset. It actually represents the pixel size in UV coordinates, for the `icon.png` image this means `vec2(0.015625, 0.015625)` or `1/64`, took me a while to figure this out... perhaps it's a better way to saying it.

**Comment by clayjohn** [comment]
What about 'Size of individual pixels in texture'? Or maybe it would be better to do a more mathematical definition like 'The inverse of the Texture resolution'?

**Comment by clayjohn** [comment]
@razcore-art its definitely accurate, but I think people who are new to graphics will have a tough time understanding what that means. What do you think about "normalized size of pixel in texture"?

**Comment by razcore-art** [comment]
@DarkEnchanter1 I think that's even more technical and it isn't really correct, cause it's not UV of texel, it's more like "normalized texel size", but still I for one did even know texels existed until now.
@clayjohn I think your definition is very similar to mine, actually I don't know why I said "UV-normalized". Ya I'd go with your definition if there's no other objections.

**Comment by mhilbrunner** [comment]
I would actually prefer the description to include something like your above example:
>for the Godot icon.png image (64x64) this means vec2(0.015625, 0.015625) or 1/64

**Comment by clayjohn** [comment]
I agree that it is much more clear to give an example. My only concern is being able to fit all that in the space provided. My understanding is that you only have the size of that little box. 

One idea would be to explain that in a sentence below the table of inputs and outputs. I think many of the terms would benefit from having a short example at the bottom as well.

**Comment by mhilbrunner** [comment]
Those table cells should expand fine with more content.

**Comment by razcore-art** [comment]
cool, then I'd just add *Example: for an image of size 64x32 pixels, `TEXTURE_PIXEL_SIZE will be vec2(1/64, 1/32)*, I don't want it to depend on specific files cause files can change and no one is gonna keep track of this stuff :)

**Comment by clayjohn** [comment]
Sounds great! Ill amend the PR.

**Comment by clayjohn** [comment]
Looks like you can't expand the table. Im getting an error while building the docs.
